apis/extension/resource.go

```diff
@@ -42,6 +42,9 @@ const (
 	GPUCore        corev1.ResourceName = ResourceDomainPrefix + "gpu-core"
 	GPUMemory      corev1.ResourceName = ResourceDomainPrefix + "gpu-memory"
 	GPUMemoryRatio corev1.ResourceName = ResourceDomainPrefix + "gpu-memory-ratio"
+
+	GpuDriver string = ResourceDomainPrefix + "gpu-driver"
```

Comment:
Keep GPU prefixes consistent with the style of existing variables.

Reply:
fixed